src/main/java/com/ericsson/ei/erqueryservice/SearchParameters.java

```diff
@@ -60,8 +65,41 @@ public void setUlt(final List<LinkType> ult) {
         this.ult = ult;
     }
 
+    /**
+     * Returns the search parameters as a json string
+     * @return String
+     * @throws IOException
+     */
+    public String getAsJsonString() throws IOException {
+        ArrayList<String> dltStringArray = convertSearchParametersToArrayList(dlt);
+        ArrayList<String> ultStringArray = convertSearchParametersToArrayList(ult);
```

Comment:
what is "dlt" amd "ult"? in the StringArray names.
Use more understanding names.

Reply:
I would love to change the names, but their names in the request are called dlt and ult and i actually don't know what they stand for ❓

Reply:
Something upstream/downstream. =)

Reply:
Emelie said that it's probably downstreamLinkType and she's is right. They are declared as List<LinkType> dlt, so it seems likely. But I don't know if that name is any more descriptive to the use and purpose of the variable.

Reply:
So the question is, is it nicer to use downstreamLinkType /upstreamLinkType or keep it as dlt/ult? I'm split in my opinion 😄 Should i keep going for dlt/ult since the REST parameters are called that way(which makes them map together) or downstreamLinkType /upstreamLinkType since they are more descriptive (but might be harder to relate to the REST parameters dlt/ult?)

Reply:
dlt and ult is bad names,, write good names or comment about what dlt and ult is. I guess you pick the best and correct way of doing this :) ,, since we have discussed this kind of things lately :)

Reply:
even if the the restpost parameter is dlt.. for making request line shorter.

Reply:
Done